### ISSUE TITLE
false postitive: use_g_autoptr_inline_cleanup

### ISSUE THREAD

**Issue by blandford** [bilelmoussaoui/gobject-lint#38]
```C
play-welcome.c:243:3: error: Consider using g_autoptr(RsvgHandle) handle to avoid manual cleanup [use_g_autoptr_inline_cleanup]
grid-render.c:467:3: error: Consider using g_autoptr(PangoContext) pango_context to avoid manual cleanup [use_g_autoptr_inline_cleanup]
```
Despite being  GObjects, neither RsvgHandle nor PangoContext have `G_DEFINE_AUTOPTR_CLEANUP_FUNC` defined. They probably should but for now they're invalid suggestions.

Also filed at https://gitlab.gnome.org/GNOME/pango/-/issues/887

**Comment by bilelmoussaoui** [comment]
Well, we can't really know that.. So the only way to not mark those as false-positive is to hardcode a list of types to ignore and i rather we do not. Closing as an upstream issue
